docs/02-about-us/background-and-history.md

```diff
@@ -2,9 +2,9 @@
 
 CivicActions, Inc. is a professional services firm delivering agile system development lifecycle services with a focus on free & open source software (FOSS). One of the things that makes us unique is our radical openness. Our largest federal clients in 2016 were the Defense Security Cooperation Agency, Department of Education, Federal Communications Commission, and through our partner Acquia, Department of Justice and Department of Treasury. Our largest Non-Profit client was Cal Poly where we re-built the Digital Democracy Platform and their DevOps pipeline.
 
-CivicActions was founded as an LLC in 2004. Our initial focus was to provide hosted FOSS group forming network technologies to progressive NGOs. We have always, since inception, prioritized clients who have a mission and tactics that we believe in. We were an early adopter of Drupal, (OWL) Open Wall Linux, and CiviCRM. Over the period between 2004-2012, we served over 175 orgs including the CRR (Center for Reproductive Rights), American Public Media, Amnesty International, and SACNAS (Society for Advancement of Chicanos/Hispanics and Native Americans in Science).
+CivicActions was founded as an LLC in 2004. Our initial focus was to provide hosted FOSS group forming network technologies to progressive NGOs. We have always, since inception, prioritized clients who have a mission and tactics that we believe in. We were an early adopter of Drupal, Openwall GNU/\*/Linux, and CiviCRM. Over the period between 2004-2012, we served over 175 orgs including the CRR (Center for Reproductive Rights), American Public Media, Amnesty International, and SACNAS (Society for Advancement of Chicanos/Hispanics and Native Americans in Science).
```

Comment:
@grugnog I like the text/corrections.
Do you really want the text to read `GNU/*/Linux` ?
Also, I'm particularly proud of our work on Democracy Now!, YouthBuild USA, Equal Justice USA, and the Women's Funding Network (well, there are many others, too.) Do we need to keep it super short?

Reply:
Heh - just looked up GNU/*/Linux at https://www.openwall.com/Owl/ - all looks good - will accept and merge